README.md

```diff
@@ -110,7 +131,16 @@ def mystery6(s)
 end
 ```
 
-- What is mystery6("goodnight moon")?
-- What is mystery6("Ada Developers Academy")?
-- What is mystery6("Hi, there!")?
+- What is mystery6("goodnight moon")? " moon goodnight"
+- What is mystery6("Ada Developers Academy")? " Academy Developers Ada"
+- What is mystery6("Hi, there!")? " there! Hi,"
 - _Added Fun: How could we make the reversal happen by letter, instead of by word (i.e. Make it so that mystery6("goodnight moon") returned "noom thgindoog")?_
+```ruby
+def mystery6(s)
+  if s == nil || s.length == 0
+    return ""
+  else
+    return s[-1] + mystery6(s[0..-2])
```

Comment:
This also could have been:

```
return mystery6(s[1..-1]) + s[0]
```

Which  does the same thing, but in other languages is more transferable (at least in Java). Both definitely work though, so whichever.